### DISCUSSION TITLE
build time > 45 minutes

### DISCUSSION THREAD

**Discussion by collegewikipedia06** [vercel/vercel#5639] [Ideas]
I am trying to deploy next.js app but its build time is > 45 minutes due to which build is failing

Please help how to solve deploy

**Comment by amityweb** [comment]
Did you solve this please? I have a site with several hundred pages in about 15 countries. We have over 5000 pages to build so it goes over 45 mins. 

I have read about incremental deployment for just an edited page for example, but what about the initial deployment, we need to deploy it all the first time. 

Thanks
